docs/en/guides/i18n.md

```diff
@@ -0,0 +1,10 @@
+# Guide
+> This section explains how to manage translations for internationalization of menu items.
+
+SkyWalking UI's internationalization translations are in the `src/locales/lang`. And the translations for the menu items are in `src/locales/lang/menus`, including `label` and `description`. The translation key of `label` which is the value of `i18nKey` is from [ui-initialized-templates/menu.yaml](https://github.com/apache/skywalking/blob/master/oap-server/server-starter/src/main/resources/ui-initialized-templates/menu.yaml).  The translation key of `description` consists of the `i18nKey` value and `_desc`. The `description` contents will be displayed in the Marketplace page. For example,
```

Comment:
```suggestion
SkyWalking UI's internationalization translations are in the `src/locales/lang`. And the translations for the menu items are in `src/locales/lang/menus`, including `label` and `description`. The translation key of `label` which is the value of `i18nKey` is from [ui-initialized-templates/menu.yaml](https://github.com/apache/skywalking/blob/master/oap-server/server-starter/src/main/resources/ui-initialized-templates/menu.yaml).  The translation key of `description` consists of the `i18nKey` value and `_desc`. The `description` contents will be displayed on the Marketplace page. For example,
```

The link should be a relative path, then, our doc build could generate a link with the current release or latest build automatically.